changedetectionio/__init__.py

```diff
@@ -42,15 +42,17 @@ def main():
     global datastore
     global app
 
-    datastore_path = None
+    datastore_path = os.environ.get('DATASTORE_PATH')
     do_cleanup = False
     host = ''
     ipv6_enabled = False
     port = os.environ.get('PORT') or 5000
     ssl_mode = False
 
+    if datastore_path is not None:
+        pass
     # On Windows, create and use a default path.
-    if os.name == 'nt':
+    elif os.name == 'nt':
```

Comment:
shouldnt this stay as `if` ?

Reply:
No, it would then not respect the `DATASTORE_PATH` defined by the user (or the default one set during container build time).